src/pages/getting-started/app-guides/app-webhooks/index.mdx

```diff
@@ -13,8 +13,8 @@ App webhooks allow you to keep track of how often your application is installed
 
 If you want to be notified about your app installation, uninstallation, and payment events, configure an App Webhooks building block in <Placeholder id="DEVELOPER_CONSOLE_URL" />. To do that, provide a URL to your backend that will receive the webhooks.
 
-<div style={{maxWidth:"700px"}}>
-  <img alt="App Webhooks in LiveChat Developer Console" src="/images/getting-started/app-guides/app-webhooks/livechat-app-webhooks.png" width="1070px" height="620px"/>
+<div style={{maxWidth:"600px"}}>
```

Comment:
I have the same concerns as for the image in App Authorization

Reply:
It is only one field to complete in this BB, so I agree that it is not crucial to have the image - the user will fully understand everything without the image